### PR TITLE
added github links and updated homepage for all npm packages

### PR DESCRIPTION
**Pull request by deadlyicon**
Add GitHub links to source code and NPM pages like https://www.npmjs.com/package/@ceramicnetwork/cli

### CODE REVIEW

File: packages/3id-did-resolver/package.json [hunk 1]
```diff
@@ -9,7 +9,14 @@
     "Data"
   ],
   "author": "Joel Torstensson <oed3933@gmail.com>",
-  "homepage": "",
+    "repository": {
```
Comment (stbrody) [inline packages/3id-did-resolver/package.json:12]: please fix the formatting/indentation throughout all the files here

Comment (deadlyicon) [reply packages/3id-did-resolver/package.json:12]: thank you! I'll fix this asap

Comment (deadlyicon) [reply packages/3id-did-resolver/package.json]: fixed

**Review by stbrody — APPROVED** [review]
LGTM.  I don't know what is "normal" to put in these fields for NPM packages, but this all seems reasonable to me.  @ukstv can you confirm this looks good?

**Comment by smrz2001** [comment]
Created a [merge PR](https://github.com/ceramicnetwork/js-ceramic/pull/2515) so tests will run.

**Pull request merged**